### ISSUE TITLE
User reported issues with Figtree hinting

### ISSUE THREAD

**Issue by rokapoz** [google/fonts#7594]
From Listnr: There are some font hinting problems visible on lower resolution monitors and smaller font sizes (14-22px) on some characters (u, n, r, h, d, p, q, g) where the stem meets the rounded part - it appears double wide in thinner font weights (300-400). This contributes to uneven feel to text flow on most used body sizes.

**Comment by chrissimpkins** [comment]
@emmamarichal are you able to reproduce this in a Windows environment?  Are we currently ttfa or manually hinting the update that you onboarded in https://github.com/google/fonts/pull/6465?

**Comment by emmamarichal** [comment]
I tested it in Word / Windows with Regular (400). It appears quite ok for me, even in a very small size (8pt).
Do you have any screenshot of we you get?

<img width="1276" alt="Screenshot 2024-04-25 at 14 25 28" src="https://github.com/google/fonts/assets/64773544/5ccfff44-f200-4de4-ad55-f9ae3eda9806">

**Comment by chrissimpkins** [comment]
Unfortunately, these reports come in anonymous with no additional information to help support our attempts to reproduce.  Let's close this as unable to reproduce and see if we happen to receive future reports with additional information to look into it.  Thanks Emma!

**Comment by HritickMallick007** [comment]
I am using Figtree as my font family for a website and for some font sizes and font weights there is a line coming in the letter. 
Has anyone faced this issue ??
Can anyone give some clarity regarding it ??
Happening for chrome and firefox ... not sure about other browsers...
Attaching a screenshot below for reference :

![Screenshot from 2024-09-27 15-12-01](https://github.com/user-attachments/assets/ebf9afd1-eb9a-49f4-849e-b5a4f7836bf8)

**Comment by MojtabaTavakkoli** [comment]
I have the same issue here, but it only happens in certain situations like when using the font size in the range between 42 - 16.
![Image](https://github.com/user-attachments/assets/83668939-7e69-48d8-80ab-c7c19d61066d)



PS: I'm am using this font in a Flutter project. Please let me know if you want me to upload the code for your further checks.

**Comment by m4rc1e** [comment]
Thanks for the report.

I can replicate the issue. The most likely cause is that several nodes in the "n" are at the same coords. Some Rasterisers don't like this.

Possible fix is to simply move the nodes.

**Comment by kenmcd** [comment]
> Thanks for the report.
> 
> I can replicate the issue. The most likely cause is that several nodes in the "n" are at the same coords. Some Rasterizers don't like this.

Could this be because different nodes are the one on top?
What I mean is there is no consistency as to which node is the top one.
If you look at both `n` and the `u` ...
Sometimes the bottom section top right node is on the top of the stack,
and the bottom section top left is on the bottom of the stack.
I would think that both top section and both bottom section in either position would be fine.
But the one on the left on the bottom of the stack,
and the one on the right on the top of the stack,
for either the upper section or the lower section,
may cause the rasterizer a problem.
And the stacking is different in the different masters.
Perhaps making it all the same across masters may make this go away.

**Comment by emmamarichal** [comment]
cc @m4rc1e @erikdkennedy


We also have an overlapping issue with the variable for this font: https://github.com/google/fonts/issues/7380 and we are working on it (several fonts are concerned). 

@chrissimpkins we can plan an update of this font once the overlapping problem is solved. We will take the opportunity to merge outlines for these kind of cases.

**Comment by m4rc1e** [comment]
> Could this be because different nodes are the one on top?

Correct. I've made an FB check to catch this, https://github.com/fonttools/fontbakery/pull/4853

**Comment by Zahra-H** [comment]
Hi, I have the same issue. Any updates on this it?
Could you let us know the approximate time it will get fixed?

**Comment by RowanAldean** [comment]
We've observed the same issue on the web for Chrome & Firefox, is there an ETA on the fix? cc @erikdkennedy?

**Comment by YewoMhango** [comment]
At our company we're encountering the same issue Figtree as well. Is there a patched version of the font somewhere?

**Comment by chrissimpkins** [comment]
@emmamarichal what is the status here?

**Comment by emmamarichal** [comment]
Yes, I just started last week! @chrissimpkins I'll try to onboard it this week or next week
